openstack/nova/values.yaml

```diff
@@ -305,7 +305,7 @@ compute:
       clone_from_snapshot: false
       full_clone_snapshots: true
       # maximum of vSphere 6.5
```

Comment:
Is this still maximum of 6.5 or now 7.0 ?

Reply:
Sounds like a trick questions :D Thanks.